lib/rack/prx_auth/token_data.rb

```diff
@@ -22,13 +24,25 @@ def user_id
       end
 
       def authorized?(resource, scope=nil)
+        return globally_authorized?(scope) if resource == WILDCARD_RESOURCE_NAME
+
+        authorized_for_resource?(resource, scope) || (scope.nil? ? false : globally_authorized?(scope))
```

Comment:
Hmm... I find the short-circuit-return plus the `||` and ternary here a bit weird to read.  Maybe some good ol' fashioned if elsif elses?

Reply:
can do.